fw/http.c

```diff
@@ -747,27 +747,35 @@ tfw_http_prep_redir(TfwHttpResp *resp, unsigned short status, TfwStr *rmark,
 	if (host.len) {
 		url.chunks[url.nchunks++] = *proto;
 		url.len += proto->len;
-
-		url.chunks[url.nchunks++] = host;
-		url.len += host.len;
+		host_val = *this_cpu_ptr(&g_buf) + SLEN(S_V_DATE) + cl_len;
+		host_len = tfw_str_to_cstr(&host, host_val, RESP_BUF_LEN - SLEN(S_V_DATE) - cl_len);
```

Comment:
Too wide line. We keep our lines not wider than 80, maybe a few characters more.

Reply:
fixed